mcs/class/System.IO.Compression/corefx/Interop.Libraries.cs

```diff
@@ -6,6 +6,10 @@ internal static partial class Interop
 {
     internal static partial class Libraries
     {
+#if WIN_PLATFORM
```

Comment:
Should this also be behind a `UNITY` define check?

Reply:
I don't know, should it? I want this it impact all Windows versions of the class libraries.

Reply:
So you want this to affect mono run at the command line, out side the context of Unity?

Reply:
I think he doesn't since he wants this to impact a sgen runtime as well.

Reply:
Yes this needs to impact sgen/command line as well